shared/utility/motion/Balance.hpp

```diff
@@ -62,9 +62,9 @@ namespace utility::motion {
         NUClear::clock::time_point lastBalanceTime;
 
     public:
-        Balancer() : lastErrorQuaternion(), lastBalanceTime() {}
+        Balancer() = default;
         void configure(const YAML::Node& config);
-        void balance(const message::motion::KinematicsModel& hip,
+        void balance(const message::motion::KinematicsModel& model,
```

Comment:
Are there other Kinematics Models other than a hip one? If so, don't rename this.

Reply:
`model` is more appropriate. Not sure why it was 'hip' in the first place.

Reply:
imo this isn't used/probably will be replaced if it ever is.. but `model` makes more sense than hip

Reply:
it would have been hip because our we don't really do kinematics for the arms and is definitely something that the balance engine wouldn't have used